index.html

```diff
@@ -2080,11 +2080,24 @@
   </span>
 </div>
 
-Whenever people have the ability to cause an [=actor=] to process less of their [=data=] or to stop
-carrying out some given set of data processing that is not essential to the service, they must be
-allowed to do so without the [=actor=] retaliating, for instance by artificially removing an
-unrelated feature, by decreasing the quality of the service, or by trying to cajole, badger, or
-trick the [=person=] into opting back into the [=processing=].
+People should be free to limit the amount of private information they share,
+to request that an actor limit the uses of data already shared,
+or that data be deleted.
+When a person chooses to deny or withdraw their permission to use their data,
+retaliation is inappropriate.
+
+It is not retaliation to terminate a service
+when data that is essential to the operation of a service is withheld.
+However, it might be retaliation if withholding data
+results in actions unrelated to the use of that data.
+Examples of retaliatory behavior include:
+
+* making the process of withdrawing consent more onerous than granting it;
+
+* using threats, badgering, or trickery ([[?dark-patterns]])
+  to encourage people to reconsider their choice; and,
```

Comment:
I appreciate the wording and adjusted tone. I believe the original text is more direct on this point. Perhaps a reword to "to encourage people to reconsider their choice and opting back into the [=processing=];"

Reply:
I dropped that for two reasons:

1. brevity
2. because I think that processing (in the general sense as well as the jargon sense) the wrong emphasis, even if that is what all the legal statutes focus on; collection is for me more important

I don't know that either is more or less direct.  The original was narrower, which is a form of directness, I suppose.

Reply:
Just for context/clarity due to my W3C position, my comment above is a non-blocking suggestion.